containers-tests/tests/map-properties.hs

```diff
@@ -820,8 +820,11 @@ test_filter = do
     filter (> "x") (fromList [(5,"a"), (3,"b")]) @?= empty
     filter (< "a") (fromList [(5,"a"), (3,"b")]) @?= empty
 
-test_filteWithKey :: Assertion
-test_filteWithKey = filterWithKey (\k _ -> k > 4) (fromList [(5,"a"), (3,"b")]) @?= singleton 5 "a"
+test_filterKeys :: Assertion
+test_filterKeys = filterKeys (> 4) (fromList [(5,"a"), (3,"b")]) @?= singleton 5 "a"
```

Comment:
This is not used, it needs to be added to the list of tests above.

```hs
, testCase "filterKeys" test_filterKeys
```

Also, it would be good to have property tests (like for IntMap).

Reply:
Ok